### ISSUE TITLE
Game is crushing after first cutscene in the Hotel View.

### ISSUE THREAD

**Issue by llwantedl** [GTAMadman/Vice-City-Rainbomizer#14]
I have followed the next steps to install the mod:
1. Installed GTA Vice City in Steam
2. Copied mod files to the main directory.
3. Replaced .exe file with 1.0 version
4. Launched the game

In game:
1. In the beginning everything works well, I am able to play near Ken office.
2. Then I'm going to Hotel View -> saving -> but not able to load the save (bug 1), which I think is related just to the mod. (Original Steam VC does not have this problem)
3. After saving I'm going to the marker, seeing a cutscene, and then having an unhandled exception - and game crashed (bug 2).

I have Windows 11. Tried compatibility modes as well, doesn't work.

**Comment by GTAMadman** [comment]
Hello,

Do you have any other mods installed alongside Rainbomizer? It is possible that this could be causing the crashes.

My other suggestion would be to ensure that you are using the original international main.scm.

If the problem persists, it would also be helpful if you could send me your save file which you are unable to load.

**Comment by llwantedl** [comment]
Hi,
No, it's just a clean version downloaded from a steam. And then I have added rainbomizer and v1.0 exe.

However, I have reinstalled GTA Vice City once again without Rainbomizer and the problem with save still persists..
So I think that's the issue with my exact version of Windows 11 + GTA Vice City. I have found out that it is a common issue, it can be fixed with some random dude dll. It is strange for a licensed steam game to have that.

I suppose that Rainbomizer issue might be related to this inconsistency, so I will close this issue.
